pkg/framework/aries/default.go

```diff
@@ -44,12 +47,17 @@ func defFrameworkOpts(frameworkOpts *Aries) error {
 	}
 
 	if frameworkOpts.storeProvider == nil {
-		storeProv, err := storeProvider()
+		newStoreProvider, err := storeProvider()
```

Comment:
nit: one more revert `storeProv` is needed. Everything else LGTM.

Reply:
fixed